packages/addons/addon-depends/multimedia-tools-depends/depends/libmediainfo/package.mk

```diff
@@ -38,4 +39,8 @@ makeinstall_target() {
   done
   cp -P .libs/* ${INSTALL}/usr/lib
```

Comment:
change `cp -P .libs/* ${INSTALL}/usr/lib` to
`cp -P .libs/libmediainfo.a ${INSTALL}/usr/lib`

Suggest making sure the commit mentions that the .so file is not copied in (but used by nextpvr)